src/Microsoft.Extensions.Logging.Abstractions/project.json

```diff
@@ -21,8 +21,9 @@
       "dependencies": {
         "System.Collections": "4.0.11-*",
         "System.Collections.Concurrent": "4.0.12-*",
-        "System.Linq": "4.1.0-*",
+        "System.Diagnostics.Debug": "4.0.11-*",
```

Comment:
Had to add this package as the auto generated resource file uses it. 
https://github.com/aspnet/Logging/pull/414/files#diff-bcd18d26801d8d13cc6eda129a3ca806R33